account_invoice_line_report/report/account_invoice_report_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_invoice_report_tree_info" model="ir.ui.view">
+        <field name="name">account.invoice.report.tree.info</field>
+        <field name="model">account.invoice.report</field>
+        <field name="arch" type="xml">
+            <tree string="Invoice Line">
+                <field name="partner_id" />
+                <field name="product_categ_id" />
+                <field name="product_id" />
+                <field name="quantity" />
+                <field name="price_average" />
+                <field name="price_subtotal" />
+                <field name="currency_id" invisible="1" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_account_invoice_report_search" model="ir.ui.view">
+        <field name="model">account.invoice.report</field>
+        <field name="inherit_id" ref="account.view_account_invoice_report_search" />
+        <field name="arch" type="xml">
+            <field name="invoice_date" position="before">
+                <field
+                    name="product_uom_id"
+                    string="Partner Tags"
+                    filter_domain="[('partner_id.commercial_partner_id.category_id', '=', self)]"
+                />
+                <field
+                    name="product_uom_id"
```

Comment:
I suppose the name should be changed for this field and above?

Reply:
Following @carlosdauden's suggestion, I ended up removing both filters.